includes/CheckFor.inc

```diff
@@ -208,4 +208,61 @@ class CheckFor {
     }
     return TRUE;
   }
+
+  /**
+   * Check file for redirects.
+   *
+   * @param object $row
+   *   A row object as delivered by migrate.
+   *
+   * @return bool
+   *   TRUE - destination URI(s).
+   *   FALSE - no detectable redirects exist in the page.
+   */
+  public static function hasHtmlRedirect($row) {
+    //Checks for meta refresh redirects. Does meta support relative urls???
+    $meta = qp($row)->find('meta[http-equiv="refresh"]')->attr("url");
```

Comment:
Dan  think this use of querypath isn't quite right.   Usually you pass html into querypath and $row is an object.  I think by the time we hit the prepare_row function the querypath will already be created.   That is one of those unwired things at the moment.  When it is connected then we can alter this call  to get it right.   

Overall this is a great stub until I can get you a test migration to run it through and you can implement some small tweaks.